buildSrc/src/main/java/com/google/firebase/gradle/ReleaseGenerator.kt

```diff
@@ -1,16 +1,16 @@
 package com.google.firebase.gradle
 
 import java.io.File
-import java.io.IOException
 import org.eclipse.jgit.api.Git
 import org.eclipse.jgit.api.ListBranchCommand
 import org.eclipse.jgit.api.errors.GitAPIException
 import org.eclipse.jgit.lib.Constants
 import org.eclipse.jgit.lib.ObjectId
-import org.eclipse.jgit.lib.Ref
 import org.gradle.api.DefaultTask
 import org.gradle.api.tasks.TaskAction
 
+data class FirebaseLibrary(val moduleNames: List<String>, val directories: List<String>)
```

Comment:
Lemme find out David hip to FP practices 👀 

OT: Good job here, cleans things up quite a bit.